test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -72,7 +78,7 @@ var _ = Describe("Integrating Helm Projects with OLM", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			By("destroying the deployed package manifests-formatted operator")
-			cleanupPkgManCmd := exec.Command(tc.BinaryName, "cleanup", projectName,
+			cleanupPkgManCmd := exec.Command(tc.BinaryName, "cleanup", fmt.Sprintf("e2e-%s", tc.TestSuffix),
```

Comment:
I don’t see how this is better than getting the project name from the directory base. We should be reading the initialized project’s config and getting the `projectName` value from there.

Reply:
We do not need to read the dir name since it is fixed info.

> We should be reading the initialized project’s config and getting the projectName value from there.

Why not the command does that?